src/Symfony/Component/ErrorHandler/ErrorRenderer/ErrorRenderer.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\ErrorHandler\ErrorRenderer;
+
+use Symfony\Component\ErrorHandler\Exception\FlattenException;
+
+/**
+ * Renders an Exception that represents a Response content.
+ *
+ * @see ErrorRendererInterface
+ */
+class ErrorRenderer
+{
+    private $renderers = [];
+
+    /**
+     * @param ErrorRendererInterface[] $renderers
+     */
+    public function __construct(array $renderers)
+    {
+        foreach ($renderers as $renderer) {
+            $this->addRenderer($renderer);
+        }
+    }
+
+    public function addRenderer(ErrorRendererInterface $renderer): void
```

Comment:
Maybe return `$this`?  `public function addRenderer(ErrorRendererInterface $renderer): self`